libMBIN/Source/NMS/GameComponents/GcMissileComponentData.cs

```diff
@@ -9,6 +9,8 @@ public class GcMissileComponentData : NMSTemplate       // size: 0x28
         [NMS(Size = 0x10)]
         /* 0x000 */ public string Explosion;
 		/* 0x010 */ public float NoTargetLife;
+        [NMS(Size = 0x4, Ignore = true)]
+        /* 0x014 */ public bool[] Padding14;
```

Comment:
should be `byte[]`

Reply:
the struct is:
{
        [NMS(Size = 0x10)]
        /* 0x000 */ public string Explosion;
        /* 0x010 */ public float NoTargetLife;
        [NMS(Size = 0x10)]
        /* 0x018 */ public string Trail;
}
Trail is missing and there is no need for Padding

Reply:
NoTargetLife is a 4 byte float at offset 0x10 and Trail is at offset 0x18.  You need the padding.